test/integration/clitest/omero_rois.py

```diff
@@ -0,0 +1,102 @@
+#
+# Copyright (C) 2019 University of Dundee & Open Microscopy Environment.
+# All rights reserved.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, write to the Free Software Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+
+# Code copied from omero-rois as I could not find a way to install it
```

Comment:
This is still the case, despite `cli-deps`? Should I give it a try?

Reply:
@joshmoore Yes, please try it. Or just check I've got it right at [55e5624](https://github.com/ome/omero-cli-zarr/pull/171/commits/55e562402738f262d15179d3cc98a0ea3bfc73cd) above (and see the corresponding test failure)

Reply:
Ok. So I can get to:
```
test/integration/clitest/test_export.py::TestRender::test_export_zarr PASSED [ 33%]
test/integration/clitest/test_export.py::TestRender::test_export_plate PASSED [ 66%]
test/integration/clitest/test_export.py::TestRender::test_export_masks PASSED [100%]
```
with the following:
 * remove the omero_rois file
 * add omero_rois in setup.py as a main dependency

The reason that `cli-deps` didn't help is that https://github.com/ome/omero-test-infra/blob/master/docker#L213-L224 doesn't run "cli deps".

Either in cli-build or py-setup we need a command that installs the test dependencies. (Or we update the `cli` target to run cli-deps or similar)

Reply:
OK, thanks for testing.
I was reluctant to add `omero_rois` as a main dependency when it's only used for setting up tests.
But can do if that's preferable to the copying over the code I needed?

I opened a PR at https://github.com/ome/omero-test-infra/pull/76 to add `run cli deps` but don't know if that helps immediately?

Reply:
Probably easiest is to just override one of the py-* commands with whatever would install the additional test requirement.